server/update_branch_test.go

```diff
@@ -51,38 +51,38 @@ func TestHandeUpdateBranch(t *testing.T) {
 	s.GithubClient.Issues = is
 
 	t.Run("random user", func(t *testing.T) {
-		*msg = MsgCommenterPermission
+		*msg = msgCommenterPermission
 
 		err := s.handleUpdateBranch(ctx, "someone", pr)
 		require.Error(t, err)
-		require.EqualError(t, ErrCommenterPermission, err.Error())
+		require.EqualError(t, err, "commenter does not have permissions")
```

Comment:
Wondering if it would be better if we matched the `source` field with *msg. That's a more cleaner way without the string matching.